odxtools/specialdata.py

```diff
@@ -122,7 +123,7 @@ def _resolve_references(self, odxlinks: OdxLinkDatabase) -> None:
         for val in self.values:
             val._resolve_references(odxlinks)
 
-def read_sdgs_from_odx(et_element: Optional[ElementTree.Element],
+def create_sdgs_from_et(et_element: Optional[ElementTree.Element],
```

Comment:
sometimes you use prefix "create", sometimes "read", and others "make", it would be better to normalize all to use same prefix.

Reply:
I agree. what do you prefer?

Reply:
All seem fine, my opinion on naming:
* "read_" prefix makes it obvious about intent, so could possibly remove the "_from_et" suffix in that case
* The "create_" prefix is a more popular choice than "make_" in Python https://docs.python.org/3/search.html?q=create_

Reply:
let's go for `create_<foo>_from_et` then: I'm not really fond of `read_` because these functions do not read in anything from storage or the network (`parse_` would probably be more appropriate IMO) and I like to keep the `_from_et` suffix because python does not really support or encourage C++-style function overloading...

Reply:
You could simplify the suffix to `_from`. This makes the the entire signature `create_<foo>_from(et_element)` well readable.

Reply:
IMO this is a *very* bad idea in python because python is not typed and such a method name could make the impression that the object to be created can be made from anything. In c++ this is more feasible because the compiler will complain about stuff that does not work...

Reply:
Fair, I see that point. No errors at compile time requires additional attention by the programmers. Nonetheless, I don't believe that `_et` contains a lot information about the actual expected type if you are not 100% familiar with all the acronyms used in the project. In doubt, one has to read the documentation in either case. 

All of this is less critical in practice, since good IDEs (such as VS Code) show previews of method signatures and thereby indicate the parameter type, which is known from the typing annotation.

In the end, this is a discussion about three characters ;-) I definitely like the overall naming convention `create_<foo>_from[_et]` and am fine with both ways.

Reply:
> [...] which is known from the typing annotation.

not currently ( in the case of the first parameter for most the constructor functions). PR is in the pipeline, though...